source/about/maximize-microsoft-investments.rst

```diff
@@ -0,0 +1,72 @@
+Maximize Your Microsoft Investments
+===================================
+
+Unlock the full potential of your Microsoft Teams, M365, and Entra ID investment with Mattermost. Designed for operational teams that require advanced customization, secure workflows, and seamless deployment on segregated networks, Mattermost enhances your existing Microsoft solutions for critical mission success.  
+
+Mattermost complements Microsoft solutions with tailored capabilities designed to meet the unique needs of high-security environments, inter-agency collaboration, and external workflows, enabling teams to maximize their Microsoft investment.
+
+The following mission-ready collaboration capabilities are available:
+
+On-Premises Skype for Business Replacement
+-------------------------------------------
+
+As Skype for Business reaches end-of-life, secure organizations require an alternative that preserves on-premises control, integrates into Microsoft workflows, and meets the security standards of air-gapped and classified environments. Cloud-first replacements like Microsoft Teams are not always viable due to network segmentation, compliance restrictions, or data sovereignty mandates.
+
+- **Preserve mission-critical communication workflows** with a self-hosted Mattermost deployment that supports :doc:`1:1 calls </collaborate/make-calls>`, :ref:`screen sharing <collaborate/make-calls:share your screen>`, and :doc:`threaded messaging </collaborate/organize-conversations>` within secure environments. See :doc:`Mattermost for M365, Teams, and Outlook </integrate/mattermost-mission-collaboration-for-m365>`.
+- **Integrate Mattermost with Microsoft tools** such as Outlook, Teams, and :doc:`Entra ID Single Sign-On </onboard/sso-entraid>` to retain user workflows while centralizing identity and access control.
```

Comment:
Should the "mission collab for m365 go in this line instead? The first bullet is more about channel based messaging, where this is about integration into suite of MS tools.